x-pack/plugins/security_solution/cypress/e2e/exceptions/shared_exception_lists_management/manage_exceptions.cy.ts

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import {
+  esArchiverLoad,
+  esArchiverUnload,
+  esArchiverResetKibana,
+} from '../../../tasks/es_archiver';
+import { getNewRule } from '../../../objects/rule';
+import { login, visitWithoutDateRange } from '../../../tasks/login';
+import { createRule, deleteCustomRule } from '../../../tasks/api_calls/rules';
+import {
+  addExceptionFlyoutItemName,
+  editException,
+  editExceptionFlyoutItemName,
+  submitEditedExceptionItem,
+  submitNewExceptionItem,
+} from '../../../tasks/exceptions';
+import { DETECTIONS_RULE_MANAGEMENT_URL, EXCEPTIONS_URL } from '../../../urls/navigation';
+
+import {
+  CONFIRM_BTN,
+  MANAGE_EXCEPTION_CREATE_BUTTON_MENU,
+  MANAGE_EXCEPTION_CREATE_BUTTON_EXCEPTION,
+  RULE_ACTION_LINK_RULE_SWITCH,
+  EXCEPTION_ITEM_VIEWER_CONTAINER,
+  EXCEPTION_CARD_ITEM_NAME,
+  EXCEPTIONS_LIST_MANAGEMENT_NAME,
+  LINK_TO_SHARED_LIST_RADIO,
+  EXCEPTION_ITEM_HEADER_ACTION_MENU,
+  EXCEPTION_ITEM_OVERFLOW_ACTION_EDIT,
+  EXECPTION_ITEM_CARD_HEADER_TITLE,
+  EXCEPTION_ITEM_OVERFLOW_ACTION_DELETE,
+  EMPTY_EXCEPTIONS_VIEWER,
+} from '../../../screens/exceptions';
+import { goToRuleDetails } from '../../../tasks/alerts_detection_rules';
+import { goToExceptionsTab } from '../../../tasks/rule_details';
+import {
+  createSharedExceptionList,
+  findSharedExceptionListItemsByName,
+  waitForExceptionsTableToBeLoaded,
+} from '../../../tasks/exceptions_table';
+
+describe('Add, edit and delete exception', () => {
+  before(() => {
+    esArchiverResetKibana();
```

Comment:
Please, check if the use of this method is extremely necessary. Thanks.

Reply:
This test needs the env to be reset before it runs